index.css

```diff
@@ -1,29 +1,93 @@
 
 body {
-  border: solid 4px green;
-  width: 400px;
-  margin-left: 700px;
-  margin-top: 100px;
+  padding: 0;
+  margin: 0;
+}
+
+.title {
+  font-size: 30px;
+  margin-left: 60px;
+}
+
+.nav-menu {
+  border: solid 2px black;
+}
+
+.navbar {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.nav-list {
+  display: flex;
+  list-style: none;
+  font-size: large;
+  margin-right: 40px;
+}
+
+.list {
+  margin-right: 20px;
+  font-size: 20px;
+}
+
+.list:hover {
+  background-color: blue;
+}
+
+.new {
+  border-left: black 3px solid;
+  border-right: black 3px solid;
+  margin-right: 20px;
+  font-size: 20px;
+}
+
+.new:hover {
+  background-color: blue;
+}
+
+.contact:hover {
+  background-color: blue;
 }
 
 button {
   box-shadow: 2px 1px 0 black;
 }
 
-h1 {
+.book-title {
   text-align: center;
   padding-bottom: 10px;
 }
 
+#book-list {
+  display: none;
+  padding-top: 100px;
+  padding-bottom: 400px;
+}
+
 #userInput {
```

Comment:
- [ ]  I kindly suggest that you ensure that the book form is centered in the page because it is centered in the design as well:

| Template Design | Your Design |
| ------------- | ------------- |
| ![Screenshot from 2022-05-27 19-54-04](https://user-images.githubusercontent.com/52934128/170775265-79ecff2f-7668-468a-a055-07cd48e047e4.png)| ![Screenshot from 2022-05-27 19-45-31](https://user-images.githubusercontent.com/52934128/170775302-bc218018-8859-452a-9fcd-1bfb8908e528.png)  |

Reply:
Well resolved 👍🏽